adapter/internal/oasparser/envoyconf/access_loggers.go

```diff
@@ -78,6 +78,13 @@ func getFileAccessLogConfigs() *config_access_logv3.AccessLog {
 		for k, v := range logConf.AccessLogs.JSONFormat {
 			logFields[k] = &structpb.Value{Kind: &structpb.Value_StringValue{StringValue: v}}
 		}
+		formattedSecondaryLogFormat := strings.ReplaceAll(logConf.AccessLogs.SecondaryLogFormat, "'", "")
```

Comment:
Shall we do the other way around?
Let's remove `SecondaryLogFormat` and read it from a new config `JSONFormatSecondary`. We can use the value of the map for Text format if needed.